doc/modules/clustering.rst

```diff
@@ -140,6 +140,11 @@ model with equal covariance per component.
 :term:`inductive` clustering methods) are not designed to be applied to new,
 unseen data.
 
+.. rubric:: Examples
```

Comment:
@adrinjalali are you ok with having the example linked here? As noted before it's not obvious that it's possible to click on the images and access examples that way. Also, we haven't been consistent with linking examples in the user guide. There are quite a few places in the user guide where a link to an example is listed under the rubric "Examples" even though you can access the respective example through clicking on the image in that section (such as here: https://github.com/scikit-learn/scikit-learn/blob/main/doc/modules/clustering.rst?plain=1#L306)

Reply:
In this case I think the link is redundant, there's a nice description with a hyperlink above here.

However, it's a good point that it's not clear the the user can click on them. I wonder if @Charlie-XIAO would have an idea of how we could improve that UX.